Kconfig

```diff
@@ -50,6 +51,13 @@ config IMAGE_BOOT_PG2
 	  that it fits AND you have an external flash programmer at hand that
 	  allows to recover. Otherwise you risk to BRICK THE IOT2050!
 
+config IMAGE_FWU_PKG
+	bool "Firmware update package for all devices"
+	help
+	  This is the official firmware update package in IOT2050-FWU-PKG-Vx.x.x.tar.xz
+	  format, which includes firmware bin, and update criteria, and builtin env,
+	  and it is for all IOT2050 devices.
```

Comment:
I still have on my to-do list to refactor the firmware build again. Upstream TI folks suggested to generate both artifacts with a single binman (U-Boot) run. Then we could just append the update package generation to the firmware build and don't have to introduce a separate config option.

Reply:
Exactly, from my perspective, the solution you mentioned will be the best way.

Reply:
@jan-kiszka I will leave this PR here until you finish your work, then I will see what I can do.